tests/defined_resource_types_spec.rb

```diff
@@ -38,13 +62,23 @@
   end
 end
 describe "Task 3:", host: :localhost do
-  it 'Create an ssh key' do
+  it 'has a working solution', :solution do
+    command("echo | ssh-keygen -t rsa -P puppet")
```

Comment:
@kjhenner This test fails if executed more than once because the `.ssh/id_rsa{.pub}` files already exist. Not a big deal, but just thought I'd mention it.